lib/mix/tasks/git_ops.check_message.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Mix.Tasks.GitOps.CheckMessage do
+  use Mix.Task
+
+  @shortdoc "Check if a file's content follows the Conventional Commits spec"
+
+  @moduledoc """
+  Receives a file path and validates if it's content follows the Conventional Commits specification.
+
+      mix git_ops.check_message <path/to/file>
+
+  Logs an error if the commit message is not parse-able.
+
+  See https://www.conventionalcommits.org/en/v1.0.0/ for more details on Conventional Commits.
+  """
+
+  alias GitOps.Commit
+  alias GitOps.Config
+
+  @doc false
+  def run([path]) do
+    message = File.read!(path)
+
+    case Commit.parse(message) do
+      {:ok, _} ->
+        :ok
+
+      :error ->
```

Comment:
I'd really love to get more informative errors here, but for now this is the only way. Great.